libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/session/MapboxTripSession.kt

```diff
@@ -451,8 +453,20 @@ internal class MapboxTripSession(
         action: (BannerInstructions) -> Unit
     ) {
         if (bannerInstructionEvent.isOccurring(progress)) {
-            ifNonNull(bannerInstructionEvent.bannerInstructions) {
-                action(it)
+            ifNonNull(bannerInstructionEvent.bannerInstructions) { bannerInstructions ->
+                val bannerView = bannerInstructions.view()
+                val bannerComponents = bannerView?.components()
+                ifNonNull(bannerComponents) { components ->
+                    components.forEachIndexed { index, component ->
+                        component.takeIf { it.type() == BannerComponents.GUIDANCE_VIEW }?.let { c ->
+                            components[index] =
+                                c.toBuilder()
+                                    .imageUrl(c.imageUrl()?.plus("&access_token=$accessToken"))
```

Comment:
We're missing a test case for this.

Reply:
Hmm, still not seeing the test, but you've mentioned we already have one @abhishek1508?